Symfony/CS/Tests/Fixer/PSR2/MultipleUseFixerTest.php

```diff
@@ -40,21 +40,22 @@ public function provideCases()
             ),
             array(
                 <<<'EOF'
-use Some, Not, PHP, Like, Use, Statement;
```

Comment:
Yeah, but we do not test T_INLINE_HTML with php-like text in other fixer tests.
(Only in some old tests of regex era)